tests/stdlib/tstrformat.nim

```diff
@@ -0,0 +1,13 @@
+discard """
```

Comment:
This test should be merged with the other test cases -- no extra test case.

Reply:
There's already way to many tests in the `runnableExamples` block - nobody is gonna read them all. If anything some of the `runnableExamples` tests should be moved to testament.

Reply:
Then could you split the test cases into interesting ones in a `runnableExamples` block and a traditional `when isMainModule` block with all the remaining `checks`? Personally I don't think they are too many because all of them actually tell me something about the behavior which is easier to show in code than in words.

I think it would be helpful to keep all the `check`s in the main module because they really help to understand the `strformat` module and if they are moved to an external tests reading the `strformat` module on its own is hard for new contributors who don't know about the external test.

Reply:
This can be done in a separate PR.